### ISSUE TITLE
Set user agent in Wikidata fetcher method

### ISSUE THREAD

**Issue by jsstevenson** [cancervariants/therapy-normalization#434]
Running wikidata fetcher elicits the following warning:

```
WARNING: Please set an user agent if you interact with a Wikibase instance from the Wikimedia Foundation.
More information in the README.md and https://meta.wikimedia.org/wiki/User-Agent_policy
```

Should be fixable in the custom wags-tails class

**Comment by korikuzma** [comment]
I believe this was fixed in #446 . Can we close?

**Comment by jsstevenson** [comment]
yep closed in #446
